packages/table/src/Table.js

```diff
@@ -0,0 +1,204 @@
+import React, { useEffect } from 'react';
+import 'core-js/stable';
```

Comment:
We can remove this import. It's better to have the app using this package import `core-js` if it's needed